django/db/backends/base/introspection.py

```diff
@@ -143,13 +141,14 @@ def get_primary_key_column(self, cursor, table_name):
         """
         Returns the name of the primary key column for the given table.
         """
-        for column in six.iteritems(self.get_indexes(cursor, table_name)):
-            if column[1]['primary_key']:
-                return column[0]
+        for constraint in self.get_constraints(cursor, table_name).values():
+            if constraint['primary_key']:
+                return constraint['columns'][0]
         return None
 
     def get_indexes(self, cursor, table_name):
         """
+        Deprecated in Django 1.11, use get_constraints instead.
```

Comment:
It's just a foolproof technique I've used in the past. Without the ability to grep for the deprecation class, there's more thinking involved when doing the removals.